Makefile

```diff
@@ -2,5 +2,5 @@ test: lint
 	cd tests && bash run.sh
 
 lint:
-	flake8 --exclude .git,__pycache__,lib/certifi,lib/idna,lib/psycopg2,lib/urllib3,lib/chardet,lib/httplib2,lib/requests,lib/yaml
+	flake8 --exclude .git,__pycache__,lib/certifi,lib/idna,lib/psycopg2,lib/urllib3,lib/chardet,lib/httplib2,lib/requests,lib/yaml,lib/m2ee
```

Comment:
Why are we excluding m2ee from being checked by flake8? This is our code, so should conform to our coding standards.

Reply:
Flake8 upgrade gives some new errors on m2ee. I excluded m2ee as it be the copy of 'pip install m2ee' but it's not (yet). I made the choice not to modify imported libraries.

Reply:
I don't think that's the right choice here. The version of m2ee used in the buildpack is divergent from the upstream version of m2ee, so reconciling the two is not something that will happen any time soon. In the meantime, any m2ee changes should conform to our coding standards.

I would prefer that we either:
a) fix m2ee to comply with this version of flake8
b) skip updating flake8 (it's a linter, so unrelated to any security concerns)

Reply:
no longer in mr. opened a ticket to move the vanilla m2ee release.